sampleBlockCacheConfig.yaml

```diff
@@ -20,6 +20,7 @@ block_cache:
   mem-size-mb: 4096
   prefetch: 80
   parallelism: 128
+  use-du: true  # Use external 'du' command for disk usage calculation
```

Comment:
if default is true and than we shouldn't be explicitly putting this flag in any of the sample configs.